reservation_units/models.py

```diff
@@ -264,7 +264,7 @@ class ReservationUnit(models.Model):
         verbose_name=_("Price unit"),
         choices=PRICE_UNITS,
         default=PRICE_UNIT_PER_HOUR,
-        help_text="Unit of the price",
+        help_text=f"Unit of the price. Possible values are {', '.join(value for value, _ in PRICE_UNITS)}.",
```

Comment:
Commented above in the serializer about `help_text`

Reply:
Actually this won't apply to Mutation documentation:
![image](https://user-images.githubusercontent.com/3908548/144565798-564a53c7-f950-42f6-af5c-e0df18e0169f.png)

So I think you **must** define the `help_text` in the serializer to get it in the mutation documentation.

Reply:
I think I'll move all the help texts to the serializers, they seem to be that way for other models. 👍

Reply:
I think the base `help_text` should be in the `model` and if the field gets re-defined/overridden in serializer then we must include the `help_text` to the serializer field.

Reply:
I've modified it so that the base help texts are in the models, and if they are redefined in the serializer, the help texts are there as well. 🙂